unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/IBackpackEditorHUDView.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public interface IBackpackEditorHUDView
```

Comment:
shouldn't you use the BaseComponentView interface here? It has already the methods Show Hide and Dispose and you are using the interface in the `BackpackEditorHUDV2ComponentView` class

Reply:
Can be useful when we complete the implementation, but don't really need it right now